protoc

```diff
@@ -14,10 +14,13 @@ protocversion=3.0.0
 protocurl="https://github.com/google/protobuf/releases/download/v${protocversion}/protoc-${protocversion}-${os}-${arch}.zip"
 
 if [ ! -f "$protocbin" ]; then
-  echo "downloading $protocbin" >&2
+  echo "downloading $protocbin"
```

Comment:
Generally, we want the output of `./protoc` to actually be the output of protoc and not to include random debugging information.  Specifically, we may want to run something like `./protoc | foo` where _foo_ knows how to parse protoc output.  These sorts of log lines would interfere with that parser.

Practically, this probably isn't an issue for protoc.  But, it's best practice to not have wrappers interfere with stdout.